client/main.lua

```diff
@@ -1,73 +1,66 @@
-if GetConvar('game_enableFlyThroughWindscreen', 'false') ~= 'true' then
-    lib.print.error('game_enableFlyThroughWindscreen is not enabled. Please add `setr game_enableFlyThroughWindscreen true` to your server.cfg for this resource to work.')
-end
-
 lib.locale()
+
 local config = require 'config.client'
-local seatbeltOn = false
-local harnessOn = false
+local playerState = LocalPlayer.state
 local speedMultiplier = config.useMPH and 2.237 or 3.6
 local minSpeeds = {
-    unbuckled = config.minSpeedUnbuckled / speedMultiplier,
-    buckled = config.minSpeedBuckled / speedMultiplier,
-    harness = config.harness.minSpeed  / speedMultiplier
+    unbuckled = config.minSpeedUnbuckled * speedMultiplier,
+    buckled = config.minSpeedBuckled * speedMultiplier,
+    harness = config.harness.minSpeed * speedMultiplier

```

Comment:
Not how that works, you take the value in kmh / mph and convert it to gta units, not the other way around.